plugins/inputs/webhooks/webhooks.go

```diff
@@ -28,14 +27,16 @@ func init() {
 }
 
 type Webhooks struct {
-	ServiceAddress string
+	ServiceAddress string `toml:"service_address"`
```

Comment:
is this a breaking change?

Reply:
I don't think so. When I added logger, I tried to add also tags as suggested here: **https://github.com/influxdata/telegraf/wiki/Review#reviewing-plugin-code**